### PR TITLE
Support Nullable field

### PR DESCRIPTION
**Pull request by rodnover55**
This PR add ability to using Nullable field.

Adds new parser that checks field to value `\N` and if it equal returns nil. Otherwise calls original parser.

### CODE REVIEW

File: dataparser.go [hunk 1]
```diff
@@ -43,6 +43,23 @@ type dateTimeParser struct {
...
 
+type nullableParser struct {
+	DataParser
+}
+
+func (p *nullableParser) Parse(s io.RuneScanner) (driver.Value, error) {
+	// Clickhouse returns `\N` string for `null` in tsv format.
+	// For checking this value we need to check first two runes in `io.RuneScanner`, but we can't reset `io.RuneScanner` after it.
+	// Copy io.RuneScanner to `bytes.Buffer` and use it twice (1st for casting to string and checking to null, 2nd for passing to original parser)
+	d := readRaw(s)
```
Comment (bgaifullin) [inline dataparser.go]: What about trying to case s into Seeker interface and avoid copying data?

**Comment by shaders** [comment]
Hi, any updates there? Would be great to have nullable fields finally supported.

**Comment by bgaifullin** [comment]
@DoubleDi  I think it can be merged and improvements may be shipped as separate commit

**Comment by DoubleDi** [comment]
Thanks for your PR

**Comment by DoubleDi** [comment]
@rodnover55  can you please actualize your current PR. Would be glad to merge it

**Comment by DoubleDi** [comment]
I've merged your commits in #81 
Thanks again

**Pull request closed without merging**